### DISCUSSION TITLE
Is it possible to control drone in Unity using RC controller?

### DISCUSSION THREAD

**Discussion by duongtran11** [microsoft/AirSim#4762] [Support Q&A]
### My question?

Can you control drone using RC controller in Unity? 

### What I'm trying to achieve

I'm trying to control drone in Unity Editor using my Taranis X7 controller.
Unity Editor informed that the joystick (the Taranis controller) has connected.
I hit play, chose Drone mode, then the drone just fall to the ground, I couldn't control it with the controller.

#### Context details

Windows 10 64bit
Airsim main branch
Taranis X7 controller connect to my PC through an USB port
Unity 2019.3.12f1
settings.json:
`{
 "SimMode" : "", 
 "SettingsVersion" : 1.2, 
 "SeeDocsAt" : "https://github.com/Microsoft/AirSim/blob/main/docs/settings.md",
 "Vehicles" : {
     "SimpleFlight" : {
       "VehicleType" : "SimpleFlight"
     }
 }
}
`

**Comment by Yurii34** [comment]
Was your problem solved?
I met the same problem.
